unittests/test_ipmi_tool.py

```diff
@@ -0,0 +1,95 @@
+#! /usr/bin/python3
```

Comment:
Please move this file to unittests/framework/utils

Reply:
done.